openssl-sys/src/handwritten/evp.rs

```diff
@@ -413,22 +413,6 @@ cfg_if! {
             pub fn EVP_PKEY_get_bits(key: *const EVP_PKEY) -> c_int;
             pub fn EVP_PKEY_get_security_bits(key: *const EVP_PKEY) -> c_int;
         }
-
```

Comment:
Should these be removed from here, or from the `openssl-sys/src/evp.rs` location?

Reply:
Here - they're transcriptions of macros from the OpenSSL headers, so they always need to be defined, even when using bindgen.

Reply:
Gotcha. LGTM then.